### PR TITLE
Redirect Home button in Dashboard to Telescope

### LINKED ISSUE #2895: Improve UX of the Home button in dashboard to go back to Telescope
A very simple UX thing we are missing. We can go from Telescope blog to the Dashboard. It isn't clear where to go back from Dashboard to Telescope. It is expected to have a home button at the top left, however, the current button leads back to the Dashboard. The current button to go to Telescope is a part of the nav and is unclear - "Telescope Production"

Either change the button to lead back to Telescope blog, or change the UI to have a more expected behavior, or be more obvious.

This is the button I mean:
image

### PR DESCRIPTION
**Pull request by tcvan0707**
## Issue This PR Addresses
Fixes #2895 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Refer to #2895, this PR will allow us to redirect to Telescope main page by clicking on the Telescope icon, where we can see the blog posts instead of opening a new Dashboard page.

## Steps to test the PR

In order to test this PR locally:
1. Add my repository as a remote: `git remote add tcvan0707 https://github.com/tcvan0707/telescope.git`
2. Fetch my branch: `git fetch tcvan0707`
3. Check out my branch that I work on this issue: `git checkout dashboard-return`
4. Change the directory to `/src/api/status`: `cd src/api/status`
5. Run `pnpm dev`
6. Go to `localhost:1111` on your browser

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by menghif** [comment]
This is not very clear. Now if I click on "Telescope Dashboard" it takes me to the Telescope app. I think a better way to do it would be to add a "Back to Telescope" button at the very bottom of the Dashboard sidebar.

**Comment by tcvan0707** [comment]
I probably need your suggestion @humphd, should we get the `Telescope Dashboard` at the very top here to redirect us to the Telescope page, or should I create another button "Back to Telescope" at the very bottom and lead us to Telescope page?

Dash Board Picture

**Comment by humphd** [comment]
I think we already have a) a back button (in the browser); b) links to all the Telescope instances.

**Comment by tcvan0707** [comment]
> I think we already have a) a back button (in the browser); b) links to all the Telescope instances.

So i guess we won't need the issue and the PR for this then?

**Review by TueeNguyen — COMMENTED** [review]
Perhaps this is not needed, user already can go to different instances of Telescope using links in `Telescope links`.

**Comment by sirinoks** [comment]
> I think we already have a) a back button (in the browser); b) links to all the Telescope instances.

This is a UX change, since the expectation of a user is to go home by clicking the logo on the top left. The button to go back currently is not clear enough - "Telescope Production" is just a part of a nav bar. There needs to either be a cleared way to (perhaps literally add this button) "Go back to Telescope", or a simpler solution would be to just change the curent button's functionality. 

We might need to change the nav structure as well, but first things first - return to home ALWAYS top left, ALWAYS separate, not just part of nav.

**Comment by sirinoks** [comment]
tcvan0707, good `Steps to test the PR` BTW, appreciate it

**Review by sirinoks — APPROVED** [review]

File: src/api/status/src/views/partials/sidebar.hbs
Comment (cindyledev) [inline src/api/status/src/views/partials/sidebar.hbs:7]: Since we have local, staging, and production to worry about, the `href` here should not point to just `https://telescope.cdot.systems/`.

- For local development, the URL should be ``,
- For staging, the URL should be `https://dev.api.telescope.cdot.systems/v1/status/`
- For prod, the URL should be `https://api.telescope.cdot.systems/v1/status/`

We had a PR #2948 land recently that did something similar to what we want here.

**Review by cindyledev — CHANGES_REQUESTED** [review]

File: src/api/status/src/views/partials/sidebar.hbs
Comment (tcvan0707) [reply src/api/status/src/views/partials/sidebar.hbs:7]: Since this is unnecessary changes, so I don't know if I can keep going or closing this one.

Comment (cindyledev) [reply src/api/status/src/views/partials/sidebar.hbs:7]: Release day is tomorrow and this doesn't seem like it's a priority for 2.7, you can probably just let it brew in PR hell and get back to it for 2.8

**Comment by aserputov** [comment]
@humphd I think we need your final opinion on this one to continue or close PR.

**Comment by humphd** [comment]
My gut is that the icon and title at the top of the navbar should go back to the "main" dashboard page for this server.  So if I'm on the Staging Dashboard, and in the Build page, clicking the icon or title would go to the main Dashboard on Staging.  Same for Production.

Getting back to Telescope is a separate thing, and already implemented via other links.

**Comment by TueeNguyen** [comment]
> My gut is that the icon and title at the top of the navbar should go back to the "main" dashboard page for this server. So if I'm on the Staging Dashboard, and in the Build page, clicking the icon or title would go to the main Dashboard on Staging. Same for Production.
> 
> Getting back to Telescope is a separate thing, and already implemented via other links.

I agree, I think the icon should take user back to Dashboard main page.

**Comment by aserputov** [comment]
@TueeNguyen so should we close this one, or ?

**Comment by TueeNguyen** [comment]
@aserputov I'd say yes
@tcvan0707 @sirinoks are you guys cool if we close this?

**Comment by tcvan0707** [comment]
> @aserputov I'd say yes
> @tcvan0707 @sirinoks are you guys cool if we close this?

I’m cool with closing. I was waiting for the final decision

**Pull request closed without merging**
